python/datasources/acs_population.py

```diff
@@ -371,8 +372,9 @@ def get_id():
         return 'ACS_POPULATION'
 
     def upload_to_gcs(self, gcs_bucket, **attrs):
+        file_diff = False
         for ingester in self._create_ingesters():
-            ingester.upload_to_gcs(gcs_bucket)
+            file_diff = file_diff or ingester.upload_to_gcs(gcs_bucket)
```

Comment:
if file_diff is True, this will not evaluate the second argument

Reply:
Good catch!